src/ColumnSelectionStep/ColumnSelectionStep.tsx

```diff
@@ -165,7 +165,7 @@ export const ColumnSelectionStep: FunctionComponent<ColumnSelectionStepProps> =
   return (
     <>
       <div className="ColumnSelectionStep notebook-step">
-        <NotebookNavAnchor step={NotebookNavStep.ColumnSelection} status={anySelected ? 'done' : 'active'} />
+        <NotebookNavAnchor step={NotebookNavStep.ColumnSelection} status='done'/>
```

Comment:
I noticed that now, when the ColumnSelection step has a "default" setting (which displays the global aggregation), this must always show as `done`, otherwise we end up with "gaps" in the visual representation of the progress through the steps.

It was hardly noticeable before the AnonParams step, but with it, it became hard to miss.